objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloQueries.java

```diff
@@ -135,19 +138,23 @@ public static void tearDown() {
             core.stop();
             core = null;
         }
-        deleteSource(vantiq);
```

Comment:
Ordering here changed just to include the `deleteSource()` in the protection-oriented `if` statement.  Deleting the directory is entirely local, so the order here has no particular importance.